engine/runtime/src/test/scala/org/enso/interpreter/test/InterpreterException.scala

```diff
@@ -3,23 +3,26 @@ package org.enso.interpreter.test
 import org.graalvm.polyglot.PolyglotException
 
 case class InterpreterException(
-  @transient polyglotException: PolyglotException
+  @transient polyglotException: PolyglotException,
+  @transient output: Option[Any]
 ) extends Throwable
     with Serializable {
-  override def getMessage: String = polyglotException.getMessage
+  override def getMessage: String =
+    polyglotException.getMessage + output.map("\n" + _.toString()).getOrElse("")
```

Comment:
This is very cool, I missed this feature a lot.

Reply:
Needed few more fixes: 9fbba8b, but yes, I believe it is going to be useful.